crates/sui-e2e-tests/tests/address_balance_tests.rs

```diff
@@ -67,7 +66,9 @@ async fn test_deposits() -> Result<(), anyhow::Error> {
         verify_accumulator_exists(child_object_resolver, recipient, 1000);
     });
 
-    Ok(())
+    // ensure that no conservation failures are detected during reconfig.
+    test_cluster.trigger_reconfiguration().await;
+    test_cluster.wait_for_epoch_all_nodes(1).await;
```

Comment:
trigger_reconfiguration() would already wait for this. It's actually unreliable to wait for a specific epoch since the epoch duration is tiny and it may have already been in epoch 2 at this point.